xfstests/mount.fuse.passthrough

```diff
@@ -2,10 +2,6 @@
 
 ulimit -n 1048576
 
-# It would be easier if we could just set SCRATCH_DEV and TEST_DEV to the source directory
```

Comment:
I think there's some confusion between SCRATCH_DEV and SCRATCH_SOURCE. What this comment says is that it would be great if we could set SCRATCH_DEV to the value that we currently store in SCRATCH_SOURCE, and get rid of SCRATCH_SOURCE completely.

Are you saying that you were able to run xfstests with this change? In this case we should get rid of the comment, but also of the SCRATCH_SOURCE variable.

Or are you saying that as-is, xfstests was not working for you unless you added the `export`? In that case we should keep the comment and add the exports.

Reply:
Ah ok, sorry, but I'm not able to extract the meaning from the current comment :) I can test/change variables or comment when I'm back from vacation in 2 weeks.

Reply:
What the comment says is that `local.config` should look like this:

```
export TEST_DEV=/mnt/src/test
export TEST_DIR=/mnt/test
export SCRATCH_DEV=/mnt/src/scratch
export SCRATCH_MNT=/mnt/scratch
export FSTYP=fuse
export FUSE_SUBTYP=.passthrough
export MOUNT_OPTIONS=""
export TEST_FS_MOUNT_OPTS=""
PASSTHROUGH_PATH=<path-to>/libfuse/build/example/passthrough_hp
```

and mount.fuse.passthrough like this:

```
#!/bin/bash
ulimit -n 1048576
exec "$PASSTHROUGH_PATH" -o fsname=$dev,allow_other "$@"
```

I believe this should work in theory and is much simpler.

Unfortunately, this causes xfstests to get very confused and nothing works (at least at the time of writing this comment).

Reply:
Can we try that when I'm back from vacation? I need to pack things and still submit some kernel patches (while I actually shouldn't be in front of my monitor ;) ).

Reply:
@bsbernd Any update on this?

Reply:
I will come back to this later on today or tomorrow, once I have sent out new atomic open kernel patches.